core/ops_builtin.rs

```diff
@@ -411,3 +417,131 @@ fn op_is_terminal(
   let handle = state.resource_table.get_handle(rid)?;
   Ok(handle.is_terminal())
 }
+
+async fn do_load_job<'s>(
+  scope: &mut v8::HandleScope<'s>,
+  module_map_rc: Rc<ModuleMap>,
+  specifier: &str,
+  code: Option<String>,
+) -> Result<ModuleId, Error> {
+  if let Some(code) = code {
+    module_map_rc
+      .new_es_module(scope, false, specifier.to_owned(), code, false, None)
+      .map_err(|e| e.into_any_error(scope, false, false))?;
+  }
+
+  let mut load = ModuleMap::load_side(module_map_rc.clone(), specifier).await?;
+
+  while let Some(load_result) = load.next().await {
+    let (request, info) = load_result?;
+    load
+      .register_and_recurse(scope, &request, info)
+      .map_err(|e| e.into_any_error(scope, false, false))?;
+  }
+
+  let root_id = load.root_module_id.expect("Root module should be loaded");
+  module_map_rc
+    .instantiate_module(scope, root_id)
+    .map_err(|e| {
+      let exception = v8::Local::new(scope, e);
+      exception_to_err_result::<()>(scope, exception, false, false).unwrap_err()
+    })?;
+
+  Ok(root_id)
+}
+
+/// Wrap module with another module that also exports `__esModule=true`.
+fn wrap_module<'s>(
+  scope: &mut v8::HandleScope<'s>,
+  module: v8::Local<'s, v8::Module>,
+) -> Option<v8::Local<'s, v8::Module>> {
+  const SOURCE: &str = "
+  export * from 'original';
+  export {default} from 'original';
```

Comment:
What is this `original` here? It looks like a placeholder that different loaded module will point to?

Reply:
the string "original" is irrelevant, it's just the module that is returned in the resolved callback.